src/sentry/migrations/0661_create_new_broken_monitor_detection_table.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 5.0.2 on 2024-03-04 22:40
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0660_fix_cron_monitor_invalid_orgs"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="MonitorEnvBrokenDetection",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("detection_timestamp", models.DateTimeField()),
+                ("user_notified_timestamp", models.DateTimeField(null=True)),
```

Comment:
How will this table be queried? Are either of these columns going to be used in conditions?

Reply:
I think based on how we want to use this table right now querying off of the monitor_incident key will be sufficient and the other columns will just be used to keep track of state. Also gonna split this out just so PR is more readable